src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -402,44 +402,44 @@ private static String getConnectionLostFileName() {
     return GameDataFileUtils.addExtension(
         "connection_lost_on_" + DateTimeFormatter.ofPattern("MMM_dd_'at'_HH_mm").format(LocalDateTime.now()));
   }
-}
-
-
-class ServerReady implements IServerReady {
-  private final CountDownLatch m_latch;
-  private final int m_clients;
 
-  ServerReady(final int waitCount) {
-    m_clients = waitCount;
-    m_latch = new CountDownLatch(m_clients);
-  }
+  static class ServerReady implements IServerReady {
```

Comment:
I tested both `ServerReady` and `IRemoteVault`.  It turns out you don't need a lobby to test these code paths--a direct-connect network game will suffice.  I didn't see any problems during my tests. :+1:

Just for future reference...  I haven't really been keeping up with the status of Docker on Windows; last time I checked you could only run a Hyper-V container for Windows Server 2016 natively.  However, you should be able to do something similar to the following:

1. Install VirtualBox (what I describe should work with any hypervisor, but I only know the details on VirtualBox)
1. Create a Linux VM and install Docker
1. Build the TripleA lobby database Docker image per the instructions I linked to above
1. Run a TripleA lobby database Docker container per the instructions I linked to above
1. Configure your VM to forward TCP port 5432 on the guest to TCP port 5432 on the host

Then when you run the TripleA lobby on your Windows host, port 5432 should get mapped to the guest VM which is then mapped to port 5432 on the Docker container, and the lobby will be connected to the Postgres database.

There might be an easier way to automate the above using the tools that come with Docker for Windows, but I'm not very familiar with them.